third_party/2/six/moves/__init__.pyi

```diff
@@ -21,28 +21,12 @@ from UserString import UserString as UserString
 from __builtin__ import xrange as xrange
 from itertools import izip as zip
 from itertools import izip_longest as zip_longest
-import __builtin__ as builtins
```

Comment:
Do we still need a `from . import builtins` here so that this module is accessible when you just do `import six`?

Reply:
Yes, that's necessary. I've added those.